conda/recipes/ucxx/conda_build_config.yaml

```diff
@@ -21,7 +21,7 @@ cmake:
 
 python:
   - 3.10
-  - 3.11
+  - 3.12
```

Comment:
Shouldn't both 3.11 and 3.12 be included here?

Reply:
ah yes this is a mistake, fixed in https://github.com/rapidsai/ucxx/pull/276/commits/7b9f38614547684cd15efbb9c5bdff209ced45a3, sorry about that